server/routes_src/index.ts

```diff
@@ -12,4 +12,19 @@ router.post("/register", async (req, res, next) => {
   res.status(200).send({success: true, password: password});
 });
 
+router.get("/notification", async (req, res, next) => {
+  const notifications = await db.getNotifications(req.body.username);
+  res.status(200).send({success: true, notifications});
+});
```

Comment:
Needs authentication
```suggestion
router.get("/notification", async (req, res, next) => {
  try {
    const user = await db.login(req.body.username, req.body.password);
    const notifications = await db.getNotifications(req.body.username);
    res.status(200).send({success: true, notifications});
  } catch (_) {
    res.status(304).send({success: false});
});
```

Reply:
oh we're actually doing that lol